adcc/backends/eri_build_helper.py

```diff
@@ -216,36 +205,36 @@ def fill_slice(self, slices, out):
         # if the backend provides Chemists' ERIs
         # <ij|kl> -> (ik|jl)
         if self.eri_notation == "chem":
-            mo_spaces = "".join(
-                np.take(np.array(mo_spaces), [0, 2, 1, 3])
-            )
+            mo_spaces = "".join(np.take(np.array(mo_spaces), [0, 2, 1, 3]))
         spin_block_str = "".join(spin_block)
-        # TODO: probably not needed in the future
-        # since libtensor will only ask for canonical blocks
         allowed, spin_symm = is_spin_allowed(spin_block_str)
         if allowed:
-            eri = self.build_eri_phys_asym_block(
-                can_block=mo_spaces, spin_symm=spin_symm
-            )
+            eri = self.build_eri_phys_asym_block(can_block=mo_spaces,
+                                                 spin_symm=spin_symm)
             out[:] = eri[comp_block_slices]
         else:
             out[:] = 0
 
     def build_eri_phys_asym_block(self, can_block=None, spin_symm=None):
-        co = self.coeffs_occ_alpha
-        cv = self.coeffs_virt_alpha
         block = can_block
         asym_block = "".join([block[i] for i in [0, 3, 2, 1]])
-        both_blocks = "{}-{}-{}-{}".format(
-            block, asym_block, str(spin_symm.pref1), str(spin_symm.pref2)
-        )
+        both_blocks = f"{block}-{asym_block}-{spin_symm.pref1}-{spin_symm.pref2}"
+        if not self.restricted:
+            both_blocks += "-" + spin_symm.transposition
```

Comment:
I made this a format-string ... since we require python 3.6 we can do this now ... helps a lot, as in here :smile: